tcms/testruns/static/testruns/js/get.js

```diff
@@ -309,17 +310,17 @@ function changeAssigneeBulk() {
     });
 }
 
-function updateExecutionText() {
-    const executions = 0 // todo: this is the list of all selected executions
-    if (!executions.length) {
-        window.alert(default_messages.alert.no_case_selected);
-        return false;
+function updateCaseText() {
+    const selected = selectedCheckboxes()
+    if ($.isEmptyObject(selected)) {
+        return false
     }
-    //todo: translations
-    executions.forEach(executionId =>
-        jsonRPC('TestExecution.update', [executionId, { 'case_text_version': 'latest' }], () => { }, sync = true)
+
+    selected.executionIds.forEach(executionId =>
+        jsonRPC('TestExecution.update', [executionId, { 'case_text_version': 'latest' }], execution => {
+            reloadRowFor(execution)
```

Comment:
you need to animate here or move the animation inside `reloadRowFor`

Reply:
I've decided on the second, because we animate that in every other place, so no need to duplicate logic